src/components/Shop/cart/OrderDetail.tsx

```
@@ -0,0 +1,123 @@
+                                {t("orders.details.products.headers.product")}
+                            </TableColumn>
+                            <TableColumn>
+                                {t("orders.details.products.headers.quantity")}
+                            </TableColumn>
+                            <TableColumn>
+                                {t("orders.details.products.headers.price")}
+                            </TableColumn>
+                        </TableHeader>
+                        <TableBody>
+                            {order?.products?.map((product) => (
+                                <TableRow>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)